src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -208,7 +208,7 @@ class TransferBalancePage extends React.Component {
                     onBackButtonPress={() => Navigation.goBack()}
                     onCloseButtonPress={() => Navigation.dismissModal(true)}
                 />
-                <View style={[styles.flex1, styles.flexBasisAuto, styles.justifyContentCenter]}>
+                <View style={[styles.flex1, styles.justifyContentCenter]}>
```

Comment:
I saw the console error below while testing and this style change fixes it.
![error](https://user-images.githubusercontent.com/22219519/208763210-d7060f0c-28b0-409c-b070-a65801de42a0.png)

Reply:
I didn't notice any UI changes during my tests. Did I miss something?

Reply:
Done! Thanks for clarifying!